Processors/68000/68000.hpp

```diff
@@ -15,6 +15,66 @@
 
 namespace CPU::MC68000 {
 
+using OperationT = unsigned int;
+
+namespace Operation {
+
+/// Indicates that the address strobe and exactly one of the data strobes are active; you can determine
+/// which by inspecting the low bit of the provided address. The RW line indicates a read.
+static constexpr OperationT SelectByte				= 1 << 0;
+// Maintenance note: this is bit 0 to reduce the cost of getting a host-endian
+// bytewise address. The assumption that it is bit 0 is also used for branchless
+// selection in a few places. See implementation of host_endian_byte_address(),
+// value8_high(), value8_low() and value16().
+
+/// Indicates that the address and both data select strobes are active.
+static constexpr OperationT SelectWord				= 1 << 1;
+
+/// If set, indicates a read. Otherwise, a write.
+static constexpr OperationT Read					= 1 << 2;
+
+// Two-bit gap deliberately left here for PermitRead/Write below.
```

Comment:
`PermitRead` is no longer 'below' in this file.

Reply:
It is here, though. Ignore the remark.

Reply:
Perhaps just move them here?

Reply:
I've kept them separate to keep a distinction between what is part of the 68000 spec and what I've just invented. I've tried to improve the commentary to explain.